packages/FileUpload/src/FileList/index.tsx

```diff
@@ -46,21 +46,21 @@ const FileList = ({
                       label="Image title"
                       onChange={handleInputChange}
                       value={name}
-                      id={index}
+                      id={`${index}-title-${file.name}`}
                       name={NAME}
                     />
                     <Input
                       label="Alternative text"
                       onChange={handleInputChange}
                       value={data.altText}
-                      id={index}
+                      id={`${index}-altText-${file.name}`}
                       name={DATA_ATTRIBUTES.ALT_TEXT}
                     />
                     <Input
                       label="Image caption"
                       onChange={handleInputChange}
                       value={data.caption}
-                      id={index}
+                      id={`${index}-caption-${file.name}`}
```

Comment:
I am not sure to use index as id, don't you think would better use uuid package instead?

Reply:
I am using index here because of the way we deal with lists of images to upload: an array of images that each have several inputs for title, caption and alt text. I added the 'file.name' as this is unique when uploading images (you can't upload two files with the same name). Do you think this will suffice?

Reply:
I think that u can upload the same file twice in blaze. I am not quiet sure about if this will be suffice to be honest.

Reply:
No, I tried and you cannot upload two files with the same name at once. Also, I can't use the uuid package because it creates a new id every time the component is rendered, therefore I can't find the index used before because it's not there anymore, the only way I found to do this is using the array's position plus the file name because it is unique.